scripts/screener/screener.runner.ts

```diff
@@ -32,7 +29,7 @@ async function scheduleScreenerBuild(
     pullRequest: buildInfo.pullRequest,
   };
 
-  const response = await fetch(SCREENER_ENDPOINT, {
+  const response = await fetch(process.env.SCREENER_ENDPOINT, {
```

Comment:
**suggestion:** usually it's a good practice to have those "globals" encapsulated. It's out of scope of this PR for sure to setup proper env vars resolution and maintain proper TS checking. for this one I'd suggest to apply following:

```ts
const environment = {

   screener: {
      /**
       *  proper JSDoc what this is etc
      **/
      endpoint: process.env.SCREENER_ENDPOINT,
      /**
       *  proper JSDoc what this is etc
      **/
      proxyEndpoint: process.env.SCREENER_PROXY_ENDPOINT
   }
}
```

ideally it would be great if we had strictly typed `process.env.` for now it's just indexed dictionary
![image](https://user-images.githubusercontent.com/1223799/111160010-fad9cb80-8599-11eb-8dd3-876c84a10506.png)

Reply:
Out of scope but important regardless, thanks for the tip 👍